app/models/concerns/subform_utilities.rb

```diff
@@ -18,6 +18,7 @@ def distinct_by_column(column)
 
   def create_on_stockit
     return if request_from_stockit?
+
```

Comment:
And the "Best Whitespace Commit of the Year Award" goes to...? :)

cc @patrixr @abulsayyad123 @swatijadhav @bharat619